osidb/tests/factories.py

```diff
@@ -97,6 +97,20 @@ class Meta:
         ]
     )
     meta_attr = factory.Dict({"test": "1"})
+    affects = factory.RelatedFactoryList(
+        "osidb.tests.factories.AffectFactory", factory_related_name="flaw", size=1
+    )
+    comments = factory.RelatedFactoryList(
+        "osidb.tests.factories.FlawCommentFactory", factory_related_name="flaw", size=1
+    )
+    meta = factory.RelatedFactoryList(
+        "osidb.tests.factories.FlawMetaFactory", factory_related_name="flaw", size=1
+    )
+    package_versions = factory.RelatedFactoryList(
+        "osidb.tests.factories.CVEv5PackageVersionsFactory",
+        factory_related_name="flaw",
+        size=1,
+    )
```

Comment:
I'm not convinced by this, as it's not representative of how the Django ORM works

Reply:
Not sure if we want it in the project but it was added to being able to use short cuts like this to generate multiple nested entities
```
# short cut
flaw1 = FlawFactory(
    is_major_incident=True,
    meta__type=FlawMeta.FlawMetaType.REQUIRES_DOC_TEXT,
    meta__meta_attr={"status": "+"},
)

# original version
flaw1 = FlawFactory.build(is_major_incident=True)
flaw1.save(raise_validation_error=False)
FlawMetaFactory(
    flaw=flaw1
    meta_attr={"status": "+"},
    type=FlawMeta.FlawMetaType.REQUIRES_DOC_TEXT,
)
```

but it also allow to get data from parent entity, for example, if we want to generate a valid major incident flaw in the tests already with the the meta_attr we could refer this info in the `FlawMetaFactory` with `factory.SelfAttribute("..flaw.is_major_incident")`

I saw the counter part of this features was already added `flaw = factory.SubFactory(FlawFactory)` and thought it would be useful for tests make it goes both ways.